draw_visualization.js

```diff
@@ -0,0 +1,13 @@
+/*jshint esversion: 6 */
+function drawVisualization(p) { 
```

Comment:
Does the Demographic's Bar look similar to : https://github.cerner.com/Igneous/igneous-smart_server/blob/master/public/cardiac-risk-app-2.0/CardiacRiskApp.png ?
Do we want to make it consistent for all our new apps by adding a demographic's css to style them the same ?